rails/app/models/water_entry.rb

```diff
@@ -2,7 +2,7 @@ class WaterEntry < ApplicationRecord
   validates :occurred_on, presence: true
   validates :amount, presence: true
   validates :amount, measured: {greater_than: 0}
-
+  validates_presence_of :amount, :occurred_on
```

Comment:
This seems redundant. There's already `validates :occurred_on, presence: true` and `validates :amount, presence: true`. This does the same thing, right?